### PR TITLE
Remove duplicate documentation page

### PR DESCRIPTION
**Pull request by Daenyth**
Keep the better one

I'd also like to get a section on the IOApp fields that can be overridden, but we can enhance that after

### CODE REVIEW

File: site-docs/sidebars.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
       "tracing",
       "thread-model",
       "schedulers",
-      "scaling-and-tuning/io-runtime-config",
```
Comment (armanbilge) [inline site-docs/sidebars.json:15]: Actually I think we want to keep this one...

File: site-docs/sidebars.json [hunk 2]
```diff
@@ -43,8 +42,8 @@
       "std/supervisor",
       "std/async-await"
     ],
-    "Scaling and Tuning":[
-      "scaling-and-tuning/system-properties"
+    "Scaling and Tuning": [
+      "scaling-and-tuning/io-runtime-config"
     ]
```
Comment (armanbilge) [inline site-docs/sidebars.json:47]: ... and remove this one :)

**Review by armanbilge — COMMENTED** [review]
Thanks for chasing this up! Just a minor thing.

File: site-docs/sidebars.json (hunk 1 above)
Comment (armanbilge) [reply site-docs/sidebars.json:15]: In fact, it can probably be moved to that same directory as the other core runtime stuff.

**Comment by vasilmkd** [comment]
Gentle ping. @Daenyth 😆

**Pull request closed without merging**